website/organizations/index.html

```diff
@@ -22,14 +22,79 @@ <h3 class="header-section">
           <span>{{ svg('envelope') }}</span> {{ _('Thunderbird for Organizations')}}
         </h3>
         <p class="font-lg tracking-wider leading-loose mb-6">
-          {{ _('Unlike the <a href="{firefox_esr}">Mozilla Firefox ESR</a>, Thunderbird does not provide a separate ESR release. This is because the <a href="/">regular Thunderbird release</a> is already an ESR release. It remains stable and on that major version for approximately one year. Only <a href="/#channel">Thunderbird Beta</a> follows the more frequent major version releases.')|f(firefox_esr=url('firefox.enterprise')) }}
+          {{ _('Thunderbird comes with a number of key benefits which make it an excellent choice for organizational deployments of any size. And it&#39;s more than just email: As an all-in-one communication platform with integrated calendar and chat, Thunderbird is all about efficiency. Whether you are a small business or organization, a government institution, or a large enterprise - Thunderbird has been designed to scale to your needs. Learn more about the <a href="#benefits">benefits of using Thunderbird in an organization.</a>')}}
+
+        <h3 class="subheader-section">
+          {{ _('Stability, efficiency, and security for your emails') }}
+        </h3>
+        <p class="font-lg tracking-wider leading-loose mb-10 mt-0 p-links-blue">
+          {{ _('Provide your team with a stable communications solution which allows them to focus on getting their work done. The <a href="/">regular Thunderbird release</a> follows the so-called <a href="{firefox_release_calendar}">Extended Support Release (ESR) schedule</a> of the Mozilla platform. So to your advantage it will remain stable and feature-stable on that major version for approximately one year. Of course, we&#39;ll always keep you on the safe side with monthly minor upgrades which may contain stability, correctness and security upgrades.')|f(firefox_release_calendar=url('firefox.release-calendar')) }}
```

Comment:
I would take out `so-called`. Makes this phrase seem dismissive and/or sarcastic.

This whole paragraph seems kind of repetitive and it's also wrong. We've done minor updates twice a month since 78, and we also have been releasing features into those minor updates...I actually think the old paragraph was better and didn't make any already-broken promises...

I'm not sure what this new paragraph is adding other than length?

Reply:
Magnus and I both didn't like that the old paragraph was diverting attention to Firefox and per old wording started out with what we do *not* have.
To address your concerns, I've simplified this paragraph, removed the broken promises, and added back the sentence about beta.

Reply:
Fair enough. I apologize if that was a bit strongly worded. I'm a fan of brevity where possible. I think the new paragraph works well!